backend/tests/restrictionTests.test.js

```diff
@@ -0,0 +1,80 @@
+//import axios from 'axios';
+var axios = require('axios')
+
+jest.mock('axios');
+
+const UserStoreURL = 'http://20.53.224.7:8082';
+const UserDBURL = 'http://20.53.224.7:8081';
+
+test('Testing add restriction', () => {
```

Comment:
Codacy has a fix for the issue: [Replace `'Testing·add·restriction'` with `"Testing·add·restriction"`](https://app.codacy.com/gh/Harvir14/RecipEZ/pullRequest?prid=9801174)
```suggestion
test("Testing add restriction", () => {
```